services/ui-src/src/components/forms/Form.tsx

```diff
@@ -254,7 +254,7 @@ const sx = {
     paddingY: "0.25rem",
     paddingTop: 0,
     // makes the blue line continuous
-    marginBottom: 0,
+    marginTop: 0,
```

Comment:
In version 7 of CMSDS, they moved to only touch margins on the top and left, so bottom no longer does anything and needs to be swapped to top.